manifests/provider/gs-vsphere/gitrepository-collection.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: collection
+  namespace: flux-giantswarm
+spec:
+  interval: 30s
+  ref:
+    branch: master
```

```suggestion
    branch: main
```
The default branch for `vmware-app-collection` is `main`.